### PR TITLE
Dev log

### PR DESCRIPTION
**Pull request by vosmerkin**

### CODE REVIEW

File: backend/src/main/java/com/messenger/backend/controllers/UserController.java [hunk 1]
```diff
@@ -22,10 +22,12 @@ public class UserController {
     @GetMapping("/getUser")    //request userinfo for logged in user
     public UserDto getUser(@RequestParam(value = "name") String name) {
         log.info("/getUser?name={}", name);
+        System.out.println("getUser " + name);
```
Comment (Vladislav-Kisliy) [inline backend/src/main/java/com/messenger/backend/controllers/UserController.java]: This line should be removed

File: backend/src/main/resources/application-local.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/messenger
```
Comment (Vladislav-Kisliy) [inline backend/src/main/resources/application-local.properties:1]: It seems to me pushing files with local settings, it isn't good approach. Please remove it

File: ui/src/main/resources/application-local.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+logging.level.=DEBUG
```
Comment (Vladislav-Kisliy) [inline ui/src/main/resources/application-local.properties]: should be removed

File: ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java [hunk 4]
```diff
@@ -52,7 +57,11 @@ public UserDto userUpdate(UserDto userDto) throws IOException, InterruptedExcept
                 .PUT(HttpRequest.BodyPublishers.ofString(JsonMapper.toJson(userDto)))
                 .build();
         HttpResponse<String> response = null;
-        response = client.send(request, HttpResponse.BodyHandlers.ofString());
+        try {
+            response = client.send(request, HttpResponse.BodyHandlers.ofString());
+        } catch (IOException e) {
+            throw new IOException("IOException to remote address " + userUpdateAddress);
+        }
```
Comment (Vladislav-Kisliy) [inline ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java]: I didn't get this solution, could you expand this?

File: backend/src/main/java/com/messenger/backend/controllers/UserController.java (hunk 1 above)
Comment (vosmerkin) [reply backend/src/main/java/com/messenger/backend/controllers/UserController.java]: done

File: backend/src/main/resources/application-local.properties (hunk 2 above)
Comment (vosmerkin) [reply backend/src/main/resources/application-local.properties:1]: Added the file to .gitignore and removed from index (git rm --cached name_of_file)

File: ui/src/main/resources/application-local.properties (hunk 3 above)
Comment (vosmerkin) [reply ui/src/main/resources/application-local.properties]: Maybe I should better also remove it from git, as it is also a local settings file?

File: ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java (hunk 4 above)
Comment (vosmerkin) [reply ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java]: I want to add a message to the exception
I guess a better way to do this is:
```
} catch (IOException e) {
    ex.Data.Add("UserMessage", "IOException to remote address " + userUpdateAddress);
    throw;
}
```

File: ui/src/main/resources/application-local.properties (hunk 3 above)
Comment (Vladislav-Kisliy) [reply ui/src/main/resources/application-local.properties]: yes, please

Comment (vosmerkin) [reply ui/src/main/resources/application-local.properties]: Removed

File: ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java (hunk 4 above)
Comment (vosmerkin) [reply ui/src/main/java/com/messenger/ui/services/HttpBackendClient.java]: Changed to 
`} catch (IOException e) {
            throw new HttpClientIOException("IOException to remote address " + roomRequestAddress);
        }`

**Review by Vladislav-Kisliy — APPROVED** [review]

**Pull request merged**